persistence/jdbc/src/test/java/org/infinispan/test/fwk/UnitTestDatabaseManager.java

```diff
@@ -81,12 +84,27 @@
                .username("sa");
    }
 
+   public static ConnectionFactoryConfigurationBuilder<?> configureUniqueConnectionFactory(AbstractJdbcStoreConfigurationBuilder<?, ?> store, PooledConnectionFactoryConfiguration customFactoryConfiguration) {
+      return store
+              .connectionPool().read(customFactoryConfiguration);
+   }
+
+   public static ConnectionFactoryConfigurationBuilder<?> configureUniqueConnectionFactory(AbstractJdbcStoreConfigurationBuilder<?, ?> store, ManagedConnectionFactoryConfiguration customFactoryConfiguration) {
```

Comment:
I don't really see any value in these `configureUniqueConnectionFactory` helper methods as the returned code is a one-liner and the methods are only called once. I think the configuration should just occur inline where required as it is one less layer of indirection.